.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
+end_of_line = lf
+charset = utf-8
```

Comment:
two files need a different charset:

test/cases/parsing/bom/bomfile*

could you change it for them to utf-8 with BOM